setup.py

```diff
@@ -2,7 +2,7 @@
 
 setuptools.setup(
     name="rasterfoundry",
-    version="1.16.2",
+    use_scm_version=True,
```

Comment:
Do you know where `setuptools_scm` gets installed? Is it being installed as part of the reference in `setup_requires`?

Reply:
>Note: projects listed in `setup_requires` will NOT be automatically installed on the system where the setup script is being run. They are simply downloaded to the `./.eggs` directory if they’re not locally available already. If you want them to be installed, as well as being available when the setup script is run, you should add them to `install_requires` **and** `setup_requires`.

https://setuptools.readthedocs.io/en/latest/setuptools.html#id9

This makes it sound like that it is automatically downloaded and made available to `setuptools`, but not installed.

Reply:
That sounds promising. My main concern is that we are installing the latest `setuptools_scm`, which could automatically update to a point where new functionality breaks our release process. It might be good to add a version constraint if we can.